packages/api-derive/test/treasury/e2e/bounties.integration.spec.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright 2017-2020 @polkadot/api authors & contributors
+// SPDX-License-Identifier: Apache-2.0
+
+import { ApiPromise, WsProvider } from '@polkadot/api';
+
+function createApi (): Promise<ApiPromise> {
+  jest.setTimeout(30000);
+  process.env.NODE_ENV = 'test';
+
+  const provider = new WsProvider('wss://kusama-rpc.polkadot.io');
```

Comment:
A while ago I actually explicitly removed all e2e tests that we had - the issue is that they rot completely. 

For instance in this case it will stop working as soon as the bounty is completed. (data gets removed fro the state).

Reply:
Sure, no problem with that. We used it to drive development -- for now it's very handy, as we don't know the types returned for example. I left it here as I saw those are skipped in the test run. But happy to remove before merging.

Reply:
Cool, no issues.